packages/wonder-blocks-form/src/components/checkbox-group.js

```diff
@@ -136,19 +132,15 @@ export default class CheckboxGroup extends React.Component<CheckboxGroupProps> {
             <StyledFieldset data-test-id={testId} style={styles.fieldset}>
                 {/* We have a View here because fieldset cannot be used with flexbox*/}
                 <View style={style}>
-                    {typeof label === "string" ? (
+                    {label && (
                         <StyledLegend style={styles.legend}>
                             <LabelMedium>{label}</LabelMedium>
```

Comment:
**thought:** Probably out of scope for this PR, but I noticed that we are rendering a `span` here, and there would probably be an opportunity here to use `<label />` instead. Again, this could be an improvement that can be done in the future.

Reply:
oh wait, this is the general `label` for the group 🤦. Please disregard this comment.

Reply:
Good idea.  I can do this in a separate PR.